packages/application-shell/src/components/navbar/navbar.js

```diff
@@ -650,7 +678,7 @@ export class DataMenu extends React.PureComponent {
 
   render() {
     return (
-      <MenuGroup level={1}>
+      <MenuGroup id="main" level={1}>
```

Comment:
What is the id used for? Tests? 👀

Reply:
No the id is used to reference the element using `aria-owns`

https://developers.google.com/web/fundamentals/accessibility/semantics-aria/aria-labels-and-relationships#aria-owns